config/routes.rb

```diff
@@ -15,7 +15,8 @@
              
   namespace :api do
     namespace :v1 do
-      resources :recipe
+      resources :recipe, :user
```

Comment:
What's the `:user` here for? I think this might interfere with signup by overwriting the user creation route.